src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -575,10 +575,10 @@ public void RefAssembly_ReferenceAssemblyAttributeAlsoInSource()
                 var reader = assembly.GetMetadataReader();
                 var attributes = reader.GetAssemblyDefinition().GetCustomAttributes();
                 AssertEx.Equal(new string[] {
-                        "MemberReference:Void System.Runtime.CompilerServices.CompilationRelaxationsAttribute.ctor(Int32)",
-                        "MemberReference:Void System.Runtime.CompilerServices.RuntimeCompatibilityAttribute.ctor()",
```

Comment:
I don't see what source changes could have led to changing these strings.

Reply:
The extra dot comes from a fix in MetadataReaderUtils (line 322).

---
In reply to: [166739245](https://github.com/dotnet/roslyn/pull/24492#discussion_r166739245) [](ancestors = 166739245)